### PR TITLE
Updated title image for all events on the Layer5 

### LINKED ISSUE #2640: [UI] Title image for all events on the Layer5 are getting out of position
**Description**
The title image for all events on the Layer5 is getting out of position. 
Below are some examples : 
1) [Project Office Hours: Service Mesh Performance at KubeCon NA 2021](https://layer5.io/community/events/project-office-hours-service-mesh-performance-at-kubecon-na-2021)
2) [KubeCon China 2021](https://layer5.io/community/events/kubecon-china-2021)
3) [DEVCONF 2022](https://layer5.io/community/events/devconf-2022)

**Expected Behavior**
The title image for all the events needs to be aligned centrally.

**Screenshots**
image

**Environment:**
- Host OS:  Windows
- Browser:  Chrome

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Akshit42-hue**
**Description**

This PR fixes #2640 

**Notes for Reviewers**
The title image for all the events aligned centrally.
Please let me know if any changes is required

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/reusecore/PageHeader/index.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ const PageHeader = ({ category, title, img, feedlink, subtitle,  author, thumbna
     <PageHeaderWrapper>
       <div className="page-header">
         { thumbnail && <div className="feature-image">
-          <Image {...thumbnail} imgStyle={{ objectFit: "contain"}} alt={title}/>
+          <Image {...thumbnail} imgStyle={{ objectFit: "contain",  display: "block",marginLeft: "auto", marginRight: "auto"}} alt={title}/>
```
Comment (Nikhil-Ladha) [inline src/reusecore/PageHeader/index.js:49]: Same as above

File: src/components/UpcomingEventCard/index.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ const UpcomingEvents = ({ data }) => {
                 <SwiperSlide key={item.id}>
                   <div className="blog-slider_item swiper-slide">
                     <div className="blog-slider_img">
-                      <Image {...item.frontmatter.thumbnail} alt={item.frontmatter.title} />
+                      <Image {...item.frontmatter.thumbnail} imgStyle ={{maxHeight: "80rem", objectFit: "cover"}} alt={item.frontmatter.title} />
```
Comment (Nikhil-Ladha) [inline src/components/UpcomingEventCard/index.js:30]: Move these changes to the respective CSS file.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

File: src/components/UpcomingEventCard/index.js (hunk 2 above)
Comment (Akshit42-hue) [reply src/components/UpcomingEventCard/index.js:30]: Ok let me correct it

File: src/reusecore/PageHeader/index.js (hunk 1 above)
Comment (Akshit42-hue) [reply src/reusecore/PageHeader/index.js:49]: In this file what changes do I need to do

Comment (Nikhil-Ladha) [reply src/reusecore/PageHeader/index.js:49]: I mean as asked in the comment below i.e, move the changes to the imported CSS file.

**Comment by Akshit42-hue** [comment]
@Nikhil-Ladha Please do have a look

**Comment by l5io** [comment]
🚀 Preview for commit 0c8f5af8ece8fb53e6b3669368e9c1fd123b01bb at: https://6228ca4f9749400d26a2334e--layer5ng.netlify.app

File: src/components/UpcomingEventCard/EventCard.style.js [hunk 3]
```diff
@@ -13,7 +13,10 @@ const UpcomingEventsWrapper = styled.div`
 		height: 39.5rem;
 		transition: all 0.3s;
 	}
-
+	.blog-slider_img{
+		max-height: 80rem;
```
Comment (Nikhil-Ladha) [inline src/components/UpcomingEventCard/EventCard.style.js:17]: You also need to provide the max-height for the `img` tag under `gatsby--image-wrapper` under this property itself.

Comment (Nikhil-Ladha) [reply src/components/UpcomingEventCard/EventCard.style.js:17]: That is:
```
.blog-slider_img {
    ......
    ......
    .gatsby-image-wrapper {
         img {
              max-height: <some-value>
          }
    }
}
```
Do check if the intermediary `.gatsby-image-wrapper` is needed or not.
Also, reduce the value of `max-height` to around `22rem`

Comment (Akshit42-hue) [reply src/components/UpcomingEventCard/EventCard.style.js:17]: I think there is no need for .gatsby-image-wrapper and seperate img tag I have check and nothing change is there

Comment (Nikhil-Ladha) [reply src/components/UpcomingEventCard/EventCard.style.js:17]: Do you mean we don't need to update the above CSS at all?
If so, try to reduce the max-height of your current CSS to `20rem` and see if the height updates (it shouldn't)

Comment (Akshit42-hue) [reply src/components/UpcomingEventCard/EventCard.style.js:17]: Yes you are right it is not updating let me correct it

Comment (Akshit42-hue) [reply src/components/UpcomingEventCard/EventCard.style.js:17]: Changes I made
.blog-slider_img{
		max-height: 30rem; 
		object-fit: cover;
		.gatsby-image-wrapper {
			img {
				 max-height: 23rem;
			 }
	   }
	}
And here is the result
Screenshot from 2022-03-10 12-38-12

Comment (Nikhil-Ladha) [reply src/components/UpcomingEventCard/EventCard.style.js:17]: Keep values for both the classes same i.e, 25rem (considering the largest image) .

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 1a2c04656065028b8bfcf2ae7eab80a9cdd84bd8 at: https://6229af6149f5625033093c4d--layer5ng.netlify.app

**Comment by Nikhil-Ladha** [comment]
So, one last issue is that the image on the individual event, workshop pages are taking up a lot of space at the top and the reason for that is the incorrect layout configuration in the [graphql query](https://github.com/layer5io/layer5/blob/ae8cdbbd6ec263d5278e870f5a4b4a9248dc8c5d/src/templates/event-single.js#L37). It should be similar to how it is in case of [blog](https://github.com/layer5io/layer5/blob/master/src/templates/blog-single.js), [book](https://github.com/layer5io/layer5/blob/master/src/templates/book-single.js).
Please update that and hopefully, that should fix the issue. Doe let me know if it doesn't.

**Comment by Akshit42-hue** [comment]
Ok, let me give a try to fix that part

**Comment by debo19** [comment]
@Akshit42-hue were you able to find a fix? Do you want to discuss this on the websites call tomorrow?

**Comment by Akshit42-hue** [comment]
Yes, I want to discuss on websites call @debo19

**Comment by l5io** [comment]
🚀 Preview for commit f9a942d43709cfeccc21f5fe8becebf186df92a1 at: https://622f39783218bb009ce63828--layer5ng.netlify.app

**Comment by leecalcote** [comment]
Looking good. Thank you, @Akshit42-hue

**Comment by leecalcote** [comment]
Good of you to join today' call to present this update.

**Comment by Akshit42-hue** [comment]
> Good of you to join today' call to present this update.

It's been a great learning session for me

**Pull request merged**